dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1261,927 +1290,2252 @@ private static void addSummaryMessages(final int headerCount, final int importab
+     * @param choosenKeyField    A string buffer to store the chosen key field.
+     * @param builder            A builder object used to accumulate search results and messages.
+     * @return A container holding the search results including contentlets, updated inodes,
+     * condition values, and multilingual status.
+     * @throws DotDataException     If there is a data-related exception during the search process.
+     * @throws DotSecurityException If the user does not have permission to access the requested
+     *                              contentlet.
+     */
+    private static SearchByKeyFieldsResult searchByKeyFields(
+            final Structure contentType,
+            final HashMap<Integer, Object> values,
+            final Map<Integer, Field> keyFields,
+            final Pair<Host, Folder> siteAndFolder,
+            final String urlValueAssetName,
+            final boolean preview,
+            final boolean sameKeyBatchInsert,
+            final long language,
+            final boolean isMultilingual,
+            final User user,
+            final int lineNumber,
+            final StringBuffer choosenKeyField,
+            final ContentletSearchResult.Builder builder
+    ) throws DotDataException, DotSecurityException {
+
+        StringBuffer query = new StringBuffer()
+                .append("+structureName:").append(contentType.getVelocityVarName())
+                .append(" +working:true +deleted:false");
+
+        StringBuilder conditionValues = new StringBuilder();
+        boolean appendSiteToQuery = false;
+        String siteFieldValue = null;
+        boolean isMultilingualResult = isMultilingual;
+
+        // Build query from key fields
+        for (Integer column : keyFields.keySet()) {
+            Field field = keyFields.get(column);
+            Object value = values.get(column);
+
+            if (!UtilMethods.isSet(value)) {
+                builder.addMessages(ValidationMessage.builder()
+                        .type(ValidationMessageType.ERROR)
+                        .message("Key field " + field.getVelocityVarName() + " is required")
+                        .lineNumber(lineNumber)
+                        .build());
+                throw new DotRuntimeException(LINE_NO + lineNumber + " key field " +
+                        field.getVelocityVarName()
+                        + " is required since it was defined as a key\n");
+            }
+
+            String processedValue = value.toString();
+            if (value instanceof Date || value instanceof Timestamp) {
+                SimpleDateFormat formatter;
+                if (field.getFieldType().equals(Field.FieldType.DATE.toString())) {
+                    processedValue = DATE_FIELD_FORMAT.format((Date) value);
+                } else if (field.getFieldType().equals(Field.FieldType.DATE_TIME.toString())) {
+                    DateFormat df = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
This could be one of those helper methods isDateTimeField() and isTimeField etc that you have up there

Reply:
Dont we have something to do this : new SimpleDateFormat("MM/dd/yyyy"); and apply format